subjects/podiumposition/README.md

```diff
@@ -0,0 +1,38 @@
+## podiumposition
+
+### Instructions
+
+A F1 race just finished and the commentator is calling the finishing positions incorrectly.
+Help to fix this before the contestants arrive at the podium by providing the commentator with the correct podium position.
```

Comment:
Sugestion: "You need to help fix this before .... "

Reply:
It is good suggestion, but I think it also works well the way it is now.